stacc/prediction.py

```
@@ -131,31 +127,49 @@ def main():
+        "-p",
+        "--pattern",
+        default="*",
+        help="A pattern to select images from the input folder. For example, you can pass '*.png' to select only png images.",  # noqa
     )
     parser.add_argument(
-        "-m", "--model", default="colonies",
+        "-m",
+        "--model",
+        default="colonies",
         help="The model to use for counting. Can either be 'colonies' for colony counting or 'cells' for cell counting."
-        " By default the model for colony counting is used."
+        " By default the model for colony counting is used.",
     )
     parser.add_argument(
-        "-o", "--output",
-        help="The output folder for saving results. If it is given a csv with the centroids of counted objects will be saved for each image."  # noqa
+        "-o",
+        "--output",
+        help="The output folder for saving results. If it is given a csv with the centroids of counted objects will be saved for each image.",
     )
     parser.add_argument("--custom_model", help="Path to a custom trained model.")
+    parser.add_argument(
+        "--custom_distance", type=int, help="Corresponding min_distance value for postprocessing the custom model."
+    )
+    parser.add_argument(
+        "--custom_threshold", type=float, help="Corresponding threshold_abs value for postprocessing the custom model."
+    )
     args = parser.parse_args()
 
     inputs = _get_inputs(args.input, args.pattern)
     print("Counting", args.model, "in", len(inputs), "image(s).")
 
     model = get_model(args.model, args.custom_model)
-    # TODO enable over-riding this for custom models.
-    min_distance, threshold_abs = get_postprocessing_parameters(args.model)
+
+    # check that all arguments are given for custom model
+    if args.custom_model and args.custom_distance is not None and args.custom_threshold is not None:
```

This can maybe lead to confusion if only one of `custom_distance` / `custom_threshold` is given. In this case the input would just be silently ignored. I think it's better to either throw a warning or an error in that case.